src/meshapi/models.py

```diff
@@ -193,3 +198,8 @@ class SectorStatus(models.TextChoices):
     ssid = models.TextField(default=None, blank=True, null=True)
 
     notes = models.TextField(default=None, blank=True, null=True)
+
+    def __str__(self):
+        if self.ssid:
+            return self.ssid
+        return f"MeshDB Sector ID {self.id}"
```

Comment:
Not sure if I agree with this logic. The omni "sectors" all have the same `Omni-wds` SSID so in that case it's not useful for telling them apart.

What about using `building.primary_nn` in addition to the SSID? I think there's good milage in the `name` and `device_name` fields also (which we have filled in for every sector)

Reply:
This might be about to get totally changed in the schema revamp anyways so maybe it's fine for now

Reply:
There's not really a "clean" way to do this so that the list view looks pleasing and it's also totally unambiguous. I think SSID will get us closest to that. Other fields like `name` and `device name` seem to be even less unique.